test/studies/diffusion/heat_2d.chpl

```diff
@@ -0,0 +1,45 @@
+// setup simulation parameters
+config const xLen : real = 2,
+             yLen : real = 2,
+             nx = 31,
+             ny = 31,
+             nt = 50,
+             sigma = 0.25,
+             nu = 0.05;
+
+const dx : real = xLen / (nx - 1),
+      dy : real = yLen / (ny - 1),
+      dt : real = sigma * dx * dy / nu;
+
+// define a 2D domain and subdomain
+const dom = {0..<nx, 0..<ny},
+      domInner : subdomain(dom) = dom.expand(-1);
+
+// define an array to hold the solution
+var u : [dom] real;
+
+// setup initial conditions
+u = 1;
```

Comment:
I'm sticking with the split initialization to keep the "array definition" conceptually separated from "defining initial conditions".

Reply:
OK.  I didn't catch this until reading the email with your response, but I think 'setup' should be 'set up' (at least, traditionally, 'setup' has been a noun and 'set up' the verb, though this may be a case of language evolution, particularly in the technical community).